src/library/Uno.Material/Styles/Controls/v2/Button.xaml

```diff
@@ -0,0 +1,798 @@
+		<Setter Property="FontWeight"
+				Value="Medium" />
+		<Setter Property="FontFamily"
+				Value="{StaticResource MaterialMediumFontFamily}" />
+		<Setter Property="CharacterSpacing"
+				Value="{StaticResource M3MaterialButtonCharacterSpacing}" />
+		<!--End: Label Large Typo-->
+
+		<Setter Property="MinHeight"
+				Value="{StaticResource M3MaterialButtonMinHeight}" />
+		<Setter Property="MinWidth"
+				Value="{StaticResource M3MaterialButtonMinWidth}" />
+		<Setter Property="BorderThickness"
+				Value="0" />
+		<Setter Property="Padding"
+				Value="{StaticResource M3MaterialButtonPadding}" />
+		<Setter Property="CornerRadius"
+				Value="{StaticResource M3MaterialButtonCornerRadius}" />
+		<Setter Property="VerticalContentAlignment"
+				Value="Center" />
+		<Setter Property="UseSystemFocusVisuals"
+				Value="True" />
+
+		<!-- Workaround for WinUI issue: https://github.com/microsoft/microsoft-ui-xaml/issues/6388 -->
+		<Setter Property="um:ControlExtensions.Icon"
+				Value="{x:Null}" />
+
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="Button">
+
+					<toolkit:ElevatedView x:Name="ElevatedView"
+										  Margin="0"
+										  Elevation="0"
+										  CornerRadius="{TemplateBinding CornerRadius}"
+										  Background="Transparent">
+
+						<VisualStateManager.VisualStateGroups>
+							<VisualStateGroup x:Name="CommonStates">
+
+								<VisualState x:Name="Normal" />
+
+								<VisualState x:Name="PointerOver">
+									<VisualState.Setters>
+										<Setter Target="StateLayer.Background"
+												Value="{StaticResource OnPrimaryHoverBrush}" />
+									</VisualState.Setters>
+								</VisualState>
+
+								<VisualState x:Name="Pressed">
+									<VisualState.Setters>
+										<Setter Target="StateLayer.Background"
+												Value="{StaticResource OnPrimaryPressedBrush}" />
+									</VisualState.Setters>
+								</VisualState>
+
+								<VisualState x:Name="Disabled">
+									<VisualState.Setters>
+										<Setter Target="Root.Background"
+												Value="{StaticResource OnSurfaceDisabledLowBrush}" />
+										<Setter Target="IconPresenter.Foreground"
+												Value="{StaticResource OnSurfaceDisabledBrush}" />
+										<Setter Target="ContentPresenter.Foreground"
+												Value="{StaticResource OnSurfaceDisabledBrush}" />
+									</VisualState.Setters>
+								</VisualState>
+							</VisualStateGroup>
+
+							<VisualStateGroup x:Name="FocusStates">
+
+								<VisualState x:Name="Focused">
+									<VisualState.Setters>
+										<Setter Target="StateLayer.Background"
+												Value="{StaticResource OnPrimaryFocusedBrush}" />
+									</VisualState.Setters>
+								</VisualState>
+
+								<VisualState x:Name="PointerFocused">
+									<VisualState.Setters>
+										<Setter Target="StateLayer.Background"
+												Value="{StaticResource OnPrimaryFocusedBrush}" />
+									</VisualState.Setters>
+								</VisualState>
```

Comment:
I believe we need to keep the PointerFocused Visual State empty to avoid the issues that this PR fixes:https://github.com/unoplatform/Uno.Themes/pull/722